src/components/assets/NativeAssetList.vue

```diff
@@ -1,121 +1,147 @@
 <template>
-  <div class="container">
-    <div class="row">
-      <span class="text--title">{{ $t('assets.assets') }}</span>
-    </div>
+  <div>
+    <div class="container">
+      <div class="row">
+        <span class="text--title">{{ $t('assets.assets') }}</span>
+      </div>
 
-    <div class="border--separator" />
+      <div class="border--separator" />
 
-    <div v-if="tokenSymbol" class="rows">
-      <div class="row row--details">
-        <div class="row__left">
-          <div class="column--currency">
-            <img
-              width="24"
-              :src="tokenSymbol === 'SDN' ? 'icons/sdn-token.png' : 'icons/astar.png'"
-              alt="sdn"
-            />
-            <div class="column--ticker">
-              <span class="text--title">{{ tokenSymbol }}</span>
-              <span class="text--label">{{
-                tokenSymbol === 'SBY' ? 'Shibuya' : currentNetwork
-              }}</span>
+      <div v-if="tokenSymbol" class="rows">
+        <div class="row row--details">
+          <div class="row__left">
+            <div class="column--currency">
+              <img
+                width="24"
+                :src="tokenSymbol === 'SDN' ? 'icons/sdn-token.png' : 'icons/astar.png'"
```

Comment:
Please avoid hard-coding site assets

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/291/commits/6bc2a7e9567dd65ab8ff30db203b284ea77a732e) commit.